src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -1173,7 +1173,8 @@ Glyphicon for success status messages
 .alert-success:before {
     font-family: 'Glyphicons Halflings';
     display: inline-block;
-    padding-right: 0.5em;
+    padding-top: .2em;
```

Comment:
@HirdayGupta see this _[screenshot]_
![peek 2017-07-18 04-29](https://user-images.githubusercontent.com/9383897/28293588-780036b2-6b72-11e7-99d5-585d03162dd3.gif)